pkg/cmd/oauth-apiserver/openapiconfig/openapiconfig.go

```diff
@@ -115,3 +116,101 @@ func DefaultOpenAPIConfig() *openapicommon.Config {
 		SecurityDefinitions: &securityDefinitions,
 	}
 }
+
+func DefaultOpenAPIV3Config() *openapicommon.OpenAPIV3Config {
+	defNamer := apiserverendpointsopenapi.NewDefinitionNamer(legacyscheme.Scheme, extensionsapiserver.Scheme, aggregatorscheme.Scheme)
```

Comment:
It looks like we could create a default config using ` genericapiserver.DefaultOpenAPIV3Config` and then overwrite:
- `spec.Info`
- `IgnorePrefixes`
- `SecuritySchemes`

WDYT ?

Reply:
Good idea, thanks for pointing out `genericapiserver.DefaultOpenAPIV3Config`!

Reply:
OK, once this PR merges can you open a new one for using `genericapiserver.DefaultOpenAPIConfig()` in `DefaultOpenAPIConfig` ?